admin_guide/high_availability.adoc

```diff
@@ -75,66 +188,119 @@ The list of command options described here are a subset that are relevant to thi
-
-****
-`$ osadm ipfailover harreporter --selector="my-ha-service=har-reporter" --virtual-ips="10.245.2.42,10.245.2.100-104,10.245.2.142,10.245.2.242" --watch-port=4242 --replicas=7 --create`
-****
+////
+
+The `osadm ipfailover` command ensures that a pod with failover runs on each of
+the nodes matching the constraints or label used. This pod uses a VRRP (Virtual
+Router Redundancy Protocol) with link:http://www.keepalived.org/[Keepalived] to ensure that the service on the
+watched port is available, and, if needed, Keepalived will automatically float
+the VIPs if the service is not available.
+
+=== Configuring a Highly-available Routing Service
+The following steps describe how to setup a highly-available router environment with IP failover:
+
+. Label the nodes for the service. This step can be optional if you run the
+service on any of the nodes in your Kubernetes cluster and use VIPs that can
+float within those nodes. This process may already exist within a complex
+cluster, in that nodes may be filtered by any constraints or requirements
+specified (e.g. nodes with SSD drives, or higher cpu/memory/disk requirements,
+etc.).
++
+The following example defines a label as router instances that are servicing
+traffic in the US west geography "ha-router=geo-us-west":
++
+====
+----
+$ osc label nodes openshift-minion-{5,6,7,8,9} "ha-router=geo-us-west"
+----
+====
+
+. Start the router with at least two replicas on nodes matching the labels used in
+the first step. The following example runs three instances:
++
+====
+----
+$ osadm router ha-router-us-west --replicas=3 --selector="ha-router=geo-us-west" --labels="ha-router=geo-us-west" --credentials="$OPENSHIFTCONFIG" --create
+----
+====
++
+Note that the above command runs fewer router replicas than available nodes, so
+that, in the chance of node failures, Kubernetes can still ensure three
+available instances until the number of available nodes labeled
+"ha-router=geo-us-west" is below three. Additionally, the router uses the host
+network as well as ports 80 and 443, so fewer number of replicas are running to
+ensure a higher Service Level Availability (SLA). If there are no constraints on
+the service being setup for failover, it is possible to target the service to
+run on one or more, or even all, of the labeled nodes.
+
+. Finally, configure the VIPs and failover for the nodes labeled with "ha-router=geo-us-west" in step one. Ensure the number of replicas match the number of nodes and that they satisfy the label setup in step one. Specify the VIPs addresses and the port number that IP failover should monitor on the desired instances:
++
+====
+----
+$ osadm ipfailover ha-router-us-west --replicas=5 --selector="ha-router=geo-us-west" --virtual-ips="10.245.2.101-105" --watch-port=80 --credentials="$OPENSHIFTCONFIG" --create
+----
+====
+
+=== Configuring a Highly-Availabile Network Service with IP Failover [[ip-failover]]
```

Comment:
s/Available/available/ (to match the title case which the routing example section in this topic correctly uses)

Also, is "with IP Failover" necessary in the title here, since it's a subsection of "Configuring IP Failover" already? I noticed the routing example section is simply "Configuring a Highly-available Routing Service", too.